src/plugins/plugins.js

```diff
@@ -58,6 +60,8 @@ define([
 
     plugins.ImportExport = ImportExport;
 
+    plugins.ImportedRootPlugin = ImportedRootPlugin;
```

Comment:
I slightly prefer `StaticRootPlugin`, as used in `plugin.js`. "Imported" makes me think that the contents of the JSON document are going to end up somewhere else (e.g. the database) whereas this plugin displays a static hierarchy.